packages/react-draggable-dialog/src/contexts/DraggableDialogContext.ts

```diff
@@ -0,0 +1,31 @@
+import * as React from 'react';
+
+export type DraggableDialogContextValue = {
+  id: string;
+  isDragging: boolean;
+  hasBeenDragged: boolean;
+  position: {
+    x: number;
+    y: number;
+  };
+};
+
+export const draggableDialogContextDefaultValue = {
```

Comment:
this shouldn't be exported right ?

```suggestion
const draggableDialogContextDefaultValue = {
```